test/parallel/test-require-deps-deprecation.js

```diff
@@ -39,6 +42,15 @@ for (const m of deprecatedModules) {
   } catch (err) {}
 }
 
+// Instead of checking require, check that resolve isn't pointing toward
+// /node/deps, as user might already have node installed with acorn in
+// require.resolve range
 for (const m of deps) {
-  assert.throws(() => { require(m); }, /^Error: Cannot find module/);
+  let path = '';
+  try {
+    path = require.resolve(m);
+  } catch ($e) {
+    assert.ok($e);
+  }
+  assert.notStrictEqual(path, m);
```

Comment:
To avoid comparing things to the empty string this could be refactored as:

```js
let path;
try {
  path = require.resolve(m);
} catch (err) {
  continue;
}
assert.notStrictEqual(path, m);
```

Nice trick with `require.resolve` though :)

Reply:
Thanks! Correcting ASAP.

All credit should go to @Trott though ;)